### PR TITLE
Disabled GPUThreadMerger tests.

### PR DESCRIPTION
**Pull request by gaaclarke**
The GPUThreadMerger tests are flakey so I'm disabling them.

Running them locally I've seen them sometimes fail in LeaseExtension and sometimes in GpuThreadMerger.

They started showing up in LUCI at commit 1eb15c12fb1ffb2ef8e09aba98e26f3dad066e25 but that PR is unrelated.  Maybe they were always flakey, maybe something shortly before 1eb15c12fb1ffb2ef8e09aba98e26f3dad066e25 made them flakey.

```
[ RUN      ] GpuThreadMerger.LeaseExtension
[FATAL:flutter/fml/message_loop_task_queues.cc(275)] Check failed: queue_locks_.count(queue_id) && queue_entries_.count(queue_id). Trying to acquire a lock on an invalid queue_id: 32

[----------] 3 tests from GpuThreadMerger
[ RUN      ] GpuThreadMerger.IsNotOnRasterizingThread
[FATAL:flutter/fml/message_loop_task_queues.cc(275)] Check failed: queue_locks_.count(queue_id) && queue_entries_.count(queue_id). Trying to acquire a lock on an invalid queue_id: 26
Abort trap: 6
```

### CODE REVIEW

**Comment by gaaclarke** [comment]
I have verified locally that I can get the flake on commit 8997f34819e7677f2af8d46a890a4358e8c7286f

```
[ RUN      ] GpuThreadMerger.IsNotOnRasterizingThread
Segmentation fault: 11
```

**Comment by dnfield** [comment]
Is this a flake, or is it a bug in @amirh 's commit?

**Comment by gaaclarke** [comment]
> Is this a flake, or is it a bug in @amirh 's commit?

Since the crash is happening in fml and fml doesn't have any dependencies.  I'd be hard pressed to believe that any commit that doesn't touch fml caused these tests to all of a sudden because flakey.

File: testing/run_tests.py [hunk 1]
```diff
@@ -95,7 +96,7 @@ def RunCCTests(build_dir, filter):
     ]
   RunEngineExecutable(build_dir, 'flow_unittests', filter, flow_flags + shuffle_flags)
 
-  RunEngineExecutable(build_dir, 'fml_unittests', filter, [ time_sensitve_test_flag ] + shuffle_flags)
+  RunEngineExecutable(build_dir, 'fml_unittests', filter, [ time_sensitve_test_flag, gpu_thread_merger_test_flag ] + shuffle_flags)
```
Comment (amirh) [inline testing/run_tests.py]: Let's add a TODO and reference an issue to fix the flake? should probably be assigned to @iskakaushik

Comment (gaaclarke) [reply testing/run_tests.py]: done: https://github.com/flutter/flutter/issues/44002

**Comment by amirh** [comment]
LGTM

**Pull request merged**